app/src/main/res/layout-land/faq_content.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.help.faq.faqItemViewModel.FAQContentViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
```

Comment:
Please make sure its 48dp atleast so that wont fail accessiblity  test as well.

Reply:
@nikitamarysolomanpvt this comment synced very late and the PR was already closed. But I think we will take care of this. Once all highfi is done, I will again check accessibility test for all UIs.